tools/wpt/tests/test_browser.py

```diff
@@ -3,13 +3,23 @@
 import logging
 import sys
 import pytest
+import inspect
 
 from tools.wpt import browser
 
 
 logger = logging.getLogger()
 
 
+def test_all_browser_abc():
+    classes = inspect.getmembers(browser)
```

Comment:
This looks like a pretty unusual test; can we document why we're enforcing no other abstract classes?

Reply:
```suggestion
    # Make sure all subclasses of Browser implement all abstract methods
    # (except some known base classes). This is a basic sanity test in case
    # we change the ABC interface of Browser as we only instantiate some
    # products in unit tests.
    classes = inspect.getmembers(browser)
```

Is that clear?